test/fast/Crowdfund.test.ts

```diff
@@ -44,7 +44,7 @@ describe("Crowdfunds", () => {
     issuer = await smock.fake("Issuer");
     marketplace = await smock.fake("Marketplace");
     fast = await smock.fake("Fast");
-    erc20 = await smock.fake("IERC20");
+    erc20 = await smock.fake("contracts/interfaces/IERC20.sol:IERC20");
```

Comment:
Wait. What?

Reply:
Yep, super annoying... example of what happens without this:

```
  4) FastDistributionsFacet
       "before all" hook in "FastDistributionsFacet":
     Error: unable to generate smock spec from contract name.
HH701: There are multiple artifacts for contract "IERC20", please use a fully qualified name.

Please replace IERC20 for one of these options wherever you are trying to read its artifact:

@openzeppelin/contracts/token/ERC20/IERC20.sol:IERC20
contracts/interfaces/IERC20.sol:IERC20
```

Reply:
Hold on. If it picked up the OZ version, it means that something more concerning is going on. It could mean that a contract you use from OZ is referencing its own ERC20 interface.
Otherwise it wouldn't get built and artifacts generated.

I would aim at having a single source of truth here - in case something changes in our ERC20 or theirs, it could lead to something nasty.

Don't worry about this one though - it's a tooling thing we can resolve in different ways.

Reply:
Handled in https://github.com/tableturn/tt-white-contracts/pull/89/commits/a4797edaffc7fbf8150635aad73bbacb2f95b5d5 for now...